packages/xl-odt-exporter/src/odt/defaultSchema/blocks.tsx

```diff
@@ -234,17 +224,21 @@ export const odtBlockMappingForDefaultSchema: BlockMapping<
   },
 
   image: async (block, exporter) => {
-    const Buffer = (await import("buffer")).Buffer;
+    const odtExporter = exporter as ODTExporter<any, any, any>;
     const blob = await exporter.resolveFile(block.props.url);
-    const arrayBuffer = await blob.arrayBuffer();
-    const base64 = Buffer.from(arrayBuffer).toString("base64");
-    //const dimensions = await getImageDimensions(blob);
+    const fileExtension = block.props.url.split(".").pop();
```

Comment:
what if there's no `.` or sth in the url?

Reply:
I'll need to detect the file extension somehow, likely with some mimetype - file extension detector. Do we have a limited subset of images to support?

Reply:
`const blob = await this.resolveFile(url);`

does `blob.type` have the correct mimetype?